packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -1657,6 +1678,28 @@ describe('cluster', () => {
     expect(stack).toCountResources('AWS::RDS::DBInstance', 2);
 
 
+  });
+
+  test('allows setting the `iamAuthentication` property for clusterFromSnapshot', () => {
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseClusterFromSnapshot(stack, 'Database', {
+      engine: DatabaseClusterEngine.aurora({ version: AuroraEngineVersion.VER_1_22_2 }),
+      instanceProps: {
+        vpc,
+      },
+      snapshotIdentifier: 'mySnapshot',
+      iamAuthentication: true,
+    });
+
+    // THEN
+    expect(stack).toHaveResourceLike('AWS::RDS::DBCluster', {
+      EnableIAMDatabaseAuthentication: true,
+    });
+
+
```

Comment:
Please remove these empty lines.

Reply:
sorry for the mistakes made. Will fix them